src/components/Problems/Problems.tsx

```diff
@@ -155,13 +157,19 @@ export const Problems: FC = () => {
           <LoginForm closeDialog={closeLoginDialog} />
         </Stack>
       </Dialog>
+
       <Stack gap={5}>
-        <Typography variant="body1" display="inline-flex">
-          Ak si nevieš poradiť s niektorou z úloh, pozri si{' '}
-          <Link variant="button2" href={`/${seminar}/ako-riesit`}>
-            pár tipov
+        <Stack direction="row" justifyContent="space-between">
+          <Typography variant="body1" display="inline-flex">
+            Ak si nevieš poradiť s niektorou z úloh, pozri si{' '}
```

Comment:
related, ale tak osobitny problem - ta medzera `' '` nerobi nic. a hover vyzera takto:
<img width="374" alt="image" src="https://github.com/user-attachments/assets/fc0a9872-94eb-4021-9554-b61c86330e68">
neviem ale z hlavy, jak to vyriesit, sirka 480 aj tak wrappuje takto 😄 
<img width="349" alt="image" src="https://github.com/user-attachments/assets/40bd7178-0bd5-4187-9bb6-f9dca452e480">
mozno to dat do toho parent Stacku a nechat ho wrappovat 🤔 `flexWrap: 'wrap'`